### DISCUSSION TITLE
[RFC] Maintenance rules

### DISCUSSION THREAD

**Discussion by dar5hak** [dar5hak/vamp#12] [Ideas]
I was talking to @colinkiama about how to make sure the project doesn't go on a hiatus despite us having busy schedules, because let's face it: life happens.

We came up with some rules of thumb we can employ if everyone agrees:
- If someone makes a PR and it's ready for review, they assign up to two people as reviewers.
- If at least one reviewer approves it within a span of two days, they merge it along with the approval.
- If no one responds within two days, the person who submitted the PR can directly merge it. Any review comments would have to be made later. GitHub allows commenting on any branch, so that's easy.

Note that these only apply to maintainers and not to other contributors.

@nahuelwexd @devnix let me know your thoughts.

**Comment by nyx-lyb3ra** [comment]
Moved to "Ideas", because "Q&A" is a special category that requires marking some post as an "answer" to the question.

I think it's fine in general. From my side I will now try to be more active and collaborating here :)
